sys/arm64/arm64/locore.S

```diff
@@ -319,6 +319,15 @@ drop_to_el1:
 	/* Don't trap to EL2 for exceptions */
 	mov	x2, #CPTR_RES1
 	msr	cptr_el2, x2
+#if __has_feature(capabilities)
```

Comment:
```suggestion

#if __has_feature(capabilities)
```

Reply:
To be honest I don't know why we actually use cvbar_el2 and celr_el2 here, for a hybrid kernel we can just write to the non-capability part...